.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2.2.0
       - name: Run unit tests
-        run: cargo test --all-features
+        run: RUST_BACKTRACE=1 cargo test --all-features
```

Comment:
If you'd like to adjust the CI config , please can you open a separate PR for this? We've intentionally not enabled `RUST_BACKTRACE` for the reasons described here:
https://github.com/heroku/procfile-cnb/pull/75

Also worth noting for integration tests it won't do what you think it does, since it won't be propagated inside the container that runs the build/test.

Reply:
Sounds good. I can't figure out why current tests are failing. I'll move my debugging logic to it's own commit so it's easier to remove.

Reply:
Tests were failing because my template was `cargo.toml.jinja` instead of `Cargo.toml.jinja`. It passes locally because mac has a case insensitive filesystem. The reason they were getting no output was because I was following a similar pattern to the other CLI method and calling process::exit() manually. Unlike a panic, it won't output all the stdout and stderr that was captured in the test.

I refactored it to instead return a Result, then I used a wrapper function to exit if an Err is returned. we call it with the wrapper function in the CLI and without it in the tests.